test/Sentry/SentryLaravelServiceProviderTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Sentry\SentryLaravel;
+
+
+class SentryLaravelServiceProviderTest extends \Orchestra\Testbench\TestCase
+
+{
+
+    protected function getEnvironmentSetUp($app)
+    {
+        $app['config']->set('sentry.dsn', 'http://public:secret@example.com/1');
+    }
+
+    protected function getPackageProviders($app)
+    {
+        return ['Sentry\SentryLaravel\SentryLaravelServiceProvider'];
+    }
+
+    protected function getPackageAliases($app)
+    {
+        return [
+            'Sentry' => 'Sentry\SentryLaravel\SentryFacade'
+        ];
+    }
+
+    public function testIsBound()
+    {
+        $this->assertEquals(app()->bound('sentry'), true);
```

Comment:
kind of silly since every test would fail if this one did

i imagine phpunit has dependencies but i didnt look